build/ci/Dockerfile.unittest

```diff
@@ -22,6 +22,7 @@ WORKDIR $WORKDIR
 COPY Makefile .
 
 RUN go get github.com/onsi/ginkgo/ginkgo@v1.16.4 \
+    && go install sigs.k8s.io/controller-runtime/tools/setup-envtest@latest \
```

Comment:
I think we should always avoid `latest`

Reply:
could we use a more readable tag?

Reply:
[those](https://pkg.go.dev/sigs.k8s.io/controller-runtime/tools/setup-envtest?tab=versions) are the tags, no?

Reply:
right, bummer